frontend/app/templates/project/show/transportation/existing-conditions/journey-to-work/map.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="sixteen wide column">
   <Transportation::JtwMap
-    @analysis={{analysis}}
+    @analysis={{project.transportationAnalysis}}
```

Comment:
We should get in the habit of using `this.project.transportationAnalysis` unless it's part of a contextual scope (
```
<Component as |ctx|>
  <ctx.scopedVariable />
</Component
```
)

Reply:
@allthesignals ahhh, yes okay! I was wondering what our style should be, since it's been inconsistent. Good to get some motivation behind one style.  Thanks!

Reply:
Yeah, it's fine - there's a template linter but it'll be a big undertaking to make it happy for the whole codebase....so we just gotta be diligent with these templates